canonical-web-and-design/ubuntu.com
# Revert /kubernetes nav

## Done

- Reverted change to "Features" nav item in /kubernetes
- Fixed the /kubernetes/features -> /kubernetes/charmed-k8s redirect

## QA

- View the [demo](https://ubuntu-com-11490.demos.haus/kubernetes)
- See that "Features" is present in the nav, not "Charmed-K8s"
- Click the link, see that you're taken to https://ubuntu-com-11490.demos.haus/kubernetes/charmed-k8s
- Also visit https://ubuntu-com-11490.demos.haus/kubernetes/features directly, see that you're redirected again to https://ubuntu-com-11490.demos.haus/kubernetes/charmed-k8s
